### PR TITLE
Update openscale readme and notebook

### PR DESCRIPTION
**Pull request by samayamadhavan**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -46,7 +51,62 @@ git clone https://github.com/IBM/monitor-ibm-cloud-pak-with-watson-openscale
...
+
+![Select deployment space](doc/source/images/selectNewDeploymentSpace.png)
+
+* Click on the `Access control` tab and then click on `Add collaborators` on the right.
+
+![Deployment space access control](doc/source/images/deploymentSpaceAccessControl.png)
+
+* Enter "admin" as a Collaborator and select the user from the drop down list. Then click on the `Add to list` button.
+
+> **NOTE:** We are adding the user that configured the machine learning instance for OpenScale monitoring. In this case, the user is the admin user.
```
Comment (scottdangelo) [inline README.md:102]: This language is for the workshops, where we are fairly certain that the user who configurred ML for OpenScale is named `Admin`. For the notebook pattern, we should probably phrase it:
"If the user is someone other than `Admin`, then that user should be added as a collaborator here."

**Review by scottdangelo — COMMENTED** [review]
Just one thing could be clarified. Other than that, it looks ready to go.

File: README.md (hunk 1 above)
Comment (samayamadhavan) [reply README.md:102]: Do you mean, instead of the **NOTE**?

Comment (scottdangelo) [reply README.md:102]: I'll just add a separate patch.

**Pull request merged**